website/project/model.py

```diff
@@ -3978,6 +3979,8 @@ def submit_for_review(self, initiated_by, meta, save=False):
         approval.save()
         self.approval = approval
         self.add_status_log(initiated_by, DraftRegistrationLog.SUBMITTED)
+        prereg_user = User.load(settings.PREREG_FILE_CHECKOUT_USER)
```

Comment:
Given that this should only ever be used inside of `checkout_files` (perhaps more aptly named `checkout_preregistered_files`), could this be loaded inside of that function instead, after a check is done to ensure that this reg is a prereg?